.github/workflows/containers.yaml

```diff
@@ -79,25 +77,27 @@ jobs:
 
       - name: Login to GitHub Packages (ghcr.io)
         if: |
-          ${{ github.event_name != 'pull_request' }} &&
-          ${{ github.ref == 'refs/heads/main' }}
+          github.event_name != 'pull_request'
         uses: redhat-actions/podman-login@v1
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      # ${{ github.repository_owner }} - Does not handle nicely with PRs
```

Comment:
Why does this need to be changed to actor, now that we're only running it on pushes where - based on your comment - github.repository_owner is correct?

Reply:
Thats a good point. Updated